frontend/components/Domain/Recipe/RecipePage/RecipePageParts/RecipePageInstructions.vue

```diff
@@ -50,11 +50,15 @@
           <BaseButton cancel @click="dialog = false"> </BaseButton>
           <v-spacer></v-spacer>
           <div class="d-flex flex-wrap justify-end">
-            <BaseButton color="info" @click="autoSetReferences">
+            <BaseButton class="mb-1" color="info" @click="autoSetReferences">
               <template #icon> {{ $globals.icons.robot }}</template>
               {{ $t("recipe.auto") }}
             </BaseButton>
-            <BaseButton class="ml-2" save @click="setIngredientIds"> </BaseButton>
+            <BaseButton class="ml-2 mb-1" save @click="setIngredientIds"> </BaseButton>
+            <BaseButton v-if="availableNextStep" class="ml-2 mb-1" @click="saveAndOpenNextLinkIngredients">
```

Comment:
I would use  class="my-1" instead of mb-1 as this keeps the button on the same plane as the cancel button and gives us a bit more room around the buttons.